src/Core/Utilities/RegionExtensions.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Bit.Core.Enums;
+using Bit.Core.Models.Data;
+
+namespace Bit.Core.Utilities
+{
+    public static class RegionExtensions
+    {
+        public static EnvironmentUrlData GetUrls(this Region region)
+        {
+            switch (region)
+            {
+                case Region.US:
+                    return EnvironmentUrlData.DefaultUS;
+                case Region.EU:
+                    return EnvironmentUrlData.DefaultEU;
+                case Region.SelfHosted:
+                default:
+                    return null;
```

Comment:
🎨 Use just `default` that will cover the `SelfHosted` as well. Apply it also on the other extensions methods of this file.
```suggestion
                default:
                    return null;
```